src/ol/source/GeoTIFF.js

```diff
@@ -123,6 +125,49 @@ function getWorkerPool() {
   return workerPool;
 }
 
+/**
+ * Replacement for GeoTIFFImage.getResolution() to handle ModelTransformation.
+ * @param {GeoTIFFImage} image The image.
+ * @param {GeoTIFFImage} [referenceImage] The reference image.
+ * @return {Array<number>} The map x and y units per pixel.
+ */
+function getResolution(image, referenceImage) {
```

Comment:
Wouldn't it make more sense to contribute this to geotiff.js?

Reply:
I agree, if https://github.com/geotiffjs/geotiff.js/pull/413 and https://github.com/geotiffjs/geotiff.js/pull/414 are accepted and included in a release then these copies could be replaced by the real thing.

Reply:
Both are now merged and released.